### PR TITLE
Web 5.8

### PR DESCRIPTION
**Pull request by alanuecker**

### CODE REVIEW

File: _guides/html5/v5_8/concepts/events.md [hunk 1]
```diff
@@ -12,34 +12,146 @@ category:
...
+| ------------ | ------------------------------------------------------ |
+| Name:        | `UIEvent.IMAGE_LOADED` / `'imageLoaded'`               |
+| Yields:      | none                                                   |
+| Description: | Fires if the user selected a new image in the library. |
+
+| Event        | Details                                                                                                         |
+| ------------ | --------------------------------------------------------------------------------------------------------------- |
+| Name:        | `UIEvent.STICKER_UPLOADED` / `'stickerUploaded'`                                                                |
+| Yields:      | the uploaded sticker (Array<UploadedSticker>)                                                                   |
+| Description: | Fires if the user uploaded one or more custom sticker. Returns the uploaded sticker and a thumbnail of `150px`. |
```
Comment (elv1n) [inline _guides/html5/v5_8/concepts/events.md:50]:
```suggestion
| Description: | Fires if the user uploaded one or more custom stickers. Returns the uploaded sticker and a thumbnail of `150px`. |
```

File: _guides/html5/v5_8/concepts/events.md [hunk 2]
```diff
@@ -12,34 +12,146 @@ category:
...
+
+| Event        | Details                                                |
+| ------------ | ------------------------------------------------------ |
+| Name:        | `UIEvent.IMAGE_LOADED` / `'imageLoaded'`               |
+| Yields:      | none                                                   |
+| Description: | Fires if the user selected a new image in the library. |
+
+| Event        | Details                                                                                                         |
+| ------------ | --------------------------------------------------------------------------------------------------------------- |
+| Name:        | `UIEvent.STICKER_UPLOADED` / `'stickerUploaded'`                                                                |
```
Comment (elv1n) [inline _guides/html5/v5_8/concepts/events.md:48]: Is it `STICKER_UPLOAD_ADD`?

File: _guides/html5/v5_8/concepts/events.md [hunk 3]
```diff
@@ -12,34 +12,146 @@ category:
...
+        identifier: "imgly_sticker_custom",
+        name: "", // fallback if the language file does not contain a name
+        thumbnailURI: "", // the path to the category thumbnail
+        items: [
+          // load the previous custom sticker from the server and add them here
+        ],
+      },
+    ],
+  },
+  custom: {
```
Comment (elv1n) [inline _guides/html5/v5_8/concepts/events.md:139]: I think `custom` doesn't need to be in the example.

File: _guides/html5/v5_8/concepts/events.md [hunk 4]
```diff
@@ -12,34 +12,146 @@ category:
...
+}).then((editor) => {
+  editor.on(UIEvent.TOOL_CHANGED, (tool) => {
+    // send the tool information to your analytics tool
+  });
+});
 ```
 
+### Save custom sticker to your own server
+
+It is possible to restore the custom sticker an user uploaded the next time they open the editor. Once an user adds a custom sticker to the editor an `STICKER_UPLOADED` event will befired and you have the chance to upload the sticker to your own server. The custom sticker will be added to the `imgly_sticker_custom` category which can be filled with the previous items on initialising the editor.
```
Comment (elv1n) [inline _guides/html5/v5_8/concepts/events.md:120]: I would rephrase it like
===
To keep custom user stickers for the next time they open the editor you need to listen for the `STICKER_UPLOAD_ADD` event and store a sticker file on your server. When a user opens the editor you need to update their stickers configuration.

**Review by elv1n — CHANGES_REQUESTED** [review]

File: _guides/html5/v5_8/concepts/events.md (hunk 1 above)
Comment (sascha) [reply _guides/html5/v5_8/concepts/events.md:50]: "... and a thumbnail with a size of `150px`" maybe rather?

File: _guides/html5/v5_8/concepts/events.md (hunk 4 above)
Comment (sascha) [inline _guides/html5/v5_8/concepts/events.md:120]:
```suggestion
It is possible to restore the custom sticker a user uploaded the next time they open the editor. Once a user adds a custom sticker to the editor an `STICKER_UPLOADED` event will be fired and you have the chance to upload the sticker to your own server. The custom sticker will be added to the `imgly_sticker_custom` category which can be filled with the previous items on initialising the editor.
```

https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

**Review by sascha — APPROVED** [review]
Just some minor comments, so approving from my side anyway

File: _guides/html5/v5_8/concepts/events.md (hunk 1 above)
Comment (alanuecker) [reply _guides/html5/v5_8/concepts/events.md:50]: Forgot to update this part of the pr after we changed the return value of this event. Now the customer will get an id, the image and the original File object

File: _guides/html5/v5_8/concepts/events.md (hunk 2 above)
Comment (alanuecker) [reply _guides/html5/v5_8/concepts/events.md:48]: yes, forgot to update this pr after we renamed the events

File: _guides/html5/v5_8/concepts/events.md (hunk 1 above)
Comment (alanuecker) [reply _guides/html5/v5_8/concepts/events.md:50]: I would replace this line with `Fires if the user uploaded one or more custom stickers. Returns an array with the id, image and original File of the sticker.`

File: _guides/html5/v5_8/concepts/events.md (hunk 3 above)
Comment (alanuecker) [reply _guides/html5/v5_8/concepts/events.md:139]: I like to keep the `custom` part here to make it easier for the customer to change the name of the category. I would not assume that they want to keep the `Custom` label if they go the extra step to restore the custom sticker and your language file docs are not that great.
